k6/rbac/create_RTBs.js

```diff
@@ -0,0 +1,424 @@
+  listProjects, getProjectById, getRandomElements
+} from "../rancher/rancher_utils.js";
+import {
+  createUser, listUsers, listRoles, listRoleTemplates,
+  listRoleBindings, listClusterRoles, listClusterRoleBindings,
+  listCRTBs, listPRTBs, deleteRoleTemplatesByPrefix, deleteUsersByPrefix,
+  createRoleTemplate, createPRTB, createCRTB,
+  deletePRTBsByDescriptionLabel, deleteCRTBsByDescriptionLabel
+} from './rbac_utils.js';
+
+// Parameters
+const vus = __ENV.VUS || 5
+const projectCount = Number(__ENV.PROJECT_COUNT) || 10
+const userCount = Number(__ENV.USER_COUNT) || 10
+// const customPRTBsPerUser = Number(__ENV.USER_COUNT) || 5
+// const customCRTBsPerUser = Number(__ENV.USER_COUNT) || 5
+
+// Option setting
+const baseUrl = __ENV.BASE_URL
+const username = __ENV.USERNAME
+const password = __ENV.PASSWORD
+
+// Option setting
+export const options = {
+  insecureSkipTLSVerify: true,
+
+  setupTimeout: '8h',
+
+  scenarios: {
+    createPRTBs: {
+      executor: 'shared-iterations',
+      exec: 'createPRTBs',
+      vus: vus,
+      iterations: projectCount,
+      maxDuration: '1h',
+    },
+    createCRTBs: {
+      executor: 'shared-iterations',
+      exec: 'createCRTBs',
+      vus: vus,
+      iterations: userCount,
+      maxDuration: '1h',
+    }
+  },
+  thresholds: {
+    http_req_failed: ['rate<=0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(99)<=500'], // 95% of requests should be below 500ms
+    checks: ['rate>0.99'], // the rate of successful checks should be higher than 99%
+  }
+}
+
+// Custom metrics
+const numRolesTrend = new Trend('num_roles');
+const numRoleTemplatesTrend = new Trend('num_role_templates');
+const numRoleBindingsTrend = new Trend('num_role_bindings');
+const numClusterRolesTrend = new Trend('num_cluster_roles');
+const numClusterRoleBindingsTrend = new Trend('num_cluster_role_bindings');
+const numCRTBsTrend = new Trend('num_crtbs');
+const numPRTBsTrend = new Trend('num_prtbs');
+
+// Test functions, in order of execution
+export function setup() {
+  // log in
+  if (!login(baseUrl, {}, username, password)) {
+    fail(`could not login to cluster`)
+  }
+  const cookies = getCookies(baseUrl)
+
+  // delete leftovers, if any
+  cleanup(cookies)
+
+  let clusterIds = getClusterIds(cookies)
+  let clusterId = getRandomElements(clusterIds, 1)[0]
+  console.log(`Utilizing Cluster with the ID ${clusterId}`)
+  let myId = getMyId(cookies)
+
+  // Create Projects, and Users
+  for (let numProjects = 0; numProjects < projectCount; numProjects++) {
+    let res = createProject(baseUrl, cookies, `Test Project ${numProjects + 1}`, clusterId, myId)
+    if (res.status !== 201) {
+      console.log("create project status: ", res.status)
+      fail("Failed to create all expected Projects")
+    }
+  }
+
+  for (let numUsers = 0; numUsers < userCount; numUsers++) {
+    let res = createUser(baseUrl, cookies, `Test User ${numUsers + 1}`, `${numUsers + 1}`, "useruseruser")
```

Comment:
should use `username` and `password` variables here instead of hard coded values.